### PR TITLE
Missing translations script and files for each language

### PR DESCRIPTION
**Pull request by brettwp**
In words the ruby script creates a "template" by removing all the "property values" from the `messages.properties` file.  It then uses this template to synchronize each theme file.  All the comments will be updated, added, etc and all the property name=value pairs are reorganized.  Thus, in theory any new translations that are submitted can be added, for example, to the bottom of the file and the script re-run.  Thus, updating both the `/theme` and `/missing-translations` files for that language.

### CODE REVIEW

**Comment by voidmain** [comment]
Very cool! I'd be in favor of stripping multi-line properties. Most editors should be more than capable of wrapping long text, and keeping things on single lines is simpler overall. Maybe we should update the scriptage to collapse multi-line properties into a single line.

Other than that, looks good to me. I wasn't clear how the comments were handled, but I could play around with it on the branch and see how that works.

**Comment by brettwp** [comment]
Comments are handled by building a `mainFileTemplate` from the English `messages.properties` file.  On L66 in the `update.rb` add a `puts(mainFileTemplate)` and you should get the idea.

**Comment by robotdan** [comment]
Is this ready to be merged?

**Comment by brettwp** [comment]
It is now.  I wanted to make a small change for merging multilines.

File: README.md [hunk 1]
```diff
@@ -70,6 +71,23 @@ theme
 ├── message_ru.properties    Russian
 ├── message_sv.properties    Swedish
 ├── message_ua.properties    Ukrainian
+missing-translations
+├── message.properties       English
```
Comment (mooreds) [inline README.md:75]: English is missing translations?

File: update.rb [hunk 2]
```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env ruby
+
+WORKING_DIR = File.dirname(__FILE__)
+
```
Comment (mooreds) [inline update.rb:4]: Would love if this had an `-h` option to self document usage.

**Review by mooreds — CHANGES_REQUESTED** [review]
Two small changes. 

Only one I'd insist on is adding a -h switch to update.rb so that when I come back to it in 6 months I can view the help message.

Otherwise, awesome!

File: README.md (hunk 1 above)
Comment (brettwp) [reply README.md:75]: Awesome!  Total copy-pasta fail!  🤦

File: update.rb (hunk 2 above)
Comment (brettwp) [reply update.rb:28]: I added some description of how to use and what to expect.  Let me know if you think we need more.

**Review by mooreds — APPROVED** [review]
LGTM!

**Pull request merged**